### PR TITLE
Set up a 'Hello Microverse' project

### PR DESCRIPTION
**Pull request by Mofuhidy**
the first mini project in Microverse. It is a small and simple HTML and CSS project
that has heading say hello Microverse!

### CODE REVIEW

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @Mofuhidy    👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ Used the correct Git Flow 👍 

## Required Changes ♻️

- [x] Kindly, the Pull Request should have a summary describing introduced changes in more detail, so it would be better to edit the current description to make your PR more Professional :pray:
   image

- [x] **Kindly**, As suggested in the project requirement, It would be better to update your README file using [this updated template](https://github.com/microverseinc/readme-template) 😉, 
#### In every Project, the README file should be descriptive:

* All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. 
  * The only **optional** section is FAQ.
  * Make sure to add the section **Future Features**, including 1-3 potential features to be added.


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -60,40 +63,48 @@ After you're finished please remove all the comments and instructions!
...
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-
```
Comment (levy002) [inline README.md:96]: - [OPTIONAL] Kindly make sure to remove the `LIVE DEMO` because your app is not deployed yet.

**Review by levy002 — APPROVED** [review]
## Hello @Mofuhidy 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- No linter error :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Mofuhidy) [reply README.md:96]: okay I will take that in my account

**Pull request merged**